### PR TITLE
Fixed dynamic pdf header and footer for worker (Fixed #2923).

### LINKED ISSUE #2923: Regression: No Header, Footer or Table Style in generated PDF
Due to [this pull request](https://github.com/OpenSlides/OpenSlides/pull/2917), PDF documents no longer contain Headers, Footers nor Table styles.

PDF in Master is now **unstable**
why has this Pull request even been approved in the first place?

@emanuelschuetze @normanjaeckel @FinnStutzenstein

### PR DESCRIPTION
**Pull request by emanuelschuetze**
This fixed header and footer in PDF.
Still open: border color and border line width of motion meta table (see TODO in motions/pdf.js)
@FinnStutzenstein Any idea for a good fix?

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
I've lloked in the documentation: We *have* to use functions to provide these styles and we *cannot* pass functions to workers. So the one possibility left is to replace the layout block and put there a placeholder like `layout: '{{placeholder-to-insert-functions-here}}'` and recursively go through the parsed object in the worker replacing all placeholders with the four functions.

Or (this is very dirty) pass the functions as strings and eval it in the worker. This is not my favorite. I would prefer splitting up this part from the pdf creation and put it in the worker.

**Comment by emanuelschuetze** [comment]
All done. 
@FinnStutzenstein Please look at my changes. Ok for you?
@normanjaeckel Please review, too.

**Comment by FinnStutzenstein** [comment]
I can't test now. If it works it's ok for me.

**Comment by FinnStutzenstein** [comment]
> Error while generating PDF file Antrag-43.pdf: Uncaught Error: Font 'PdfFont' in style 'bold' is not defined in the font section of the document definition.

It's one of the motions you have send me.

@emanuelschuetze Not ok.

**Comment by emanuelschuetze** [comment]
As discussed with @FinnStutzenstein : I'ts not a bug, just forgot to run 'npm install' to update worker files. I'll add the gulp worker task to gulp watch.

**Comment by emanuelschuetze** [comment]
gulp watch task updated (now it watches also for changes in pdf-worker.js).

File: gulpfile.js [hunk 1]
```diff
@@ -228,9 +228,13 @@ gulp.task('default', [
  */
 
 // Watches changes in JavaScript and templates.
-gulp.task('watch', ['js', 'templates'], function () {
-    gulp.watch(path.join('openslides', '*', 'static', 'js', '**', '*.js'), ['js']);
+gulp.task('watch', ['js', 'templates', 'pdf-worker'], function () {
+    gulp.watch([
+        path.join('openslides', '*', 'static', 'js', '**', '*.js'),
+        '!' + path.join('openslides', 'core', 'static', 'js', 'core', 'pdf-worker.js')
+    ], ['js']),
```
Comment (FinnStutzenstein) [inline gulpfile.js]: The last comma has to be a semicolon. This is the Travis error.

**Pull request merged**